wiki/Article_Style_Guide/en.md

```diff
@@ -3,28 +3,53 @@
 The Article Style Guide serves as a way to help keep consistency in clarity, formatting, and layout between all articles of the osu!wiki.
 
 All English articles should use plain English.
-Your word choice should be able to explain such things in layman's terms (basically ask yourself, "If the reader is reading this, would they have to lookup any of the words?").
+Your word choice should be able to explain such things in layman's terms.
+In other words, you should ask yourself, "If the reader is reading this, would they have to lookup any of the words?"
 
 All articles must have proper grammar, correct spelling, and correct information.
-Know that reviewers may ask for changes in your pull request for blunders or suggestions.
-A good osu!wiki writer should read these reviews to help improve the overall quality of these articles to ensure optimal experience for an osu!wiki reader.
+Keep in mind that reviewers may ask for changes in your pull request for blunders or suggestions.
```

Comment:
L10: Rephrase. Sugested `` Keep in mind that reviewers would ask for changes in your pull request for blunders or suggestions.``